get-started/quick-start.mdx

```diff
@@ -102,6 +102,13 @@ It'll look something this:
   these surface areas.
 </Note>
 
+If you are comfortable with the browser inspector and need to check that the pixel is setup,
```

Comment:
Can we add to the bottom in a collapsing troubleshooting section? I think we have other examples in the docs

Reply:
Done